src/main/java/bob/Parser.java

```diff
@@ -0,0 +1,182 @@
+package bob;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import bob.command.AddDeadlineCommand;
+import bob.command.AddEventCommand;
+import bob.command.AddTodoCommand;
+import bob.command.Command;
+import bob.command.DeleteCommand;
+import bob.command.ExitCommand;
+import bob.command.ListCommand;
+import bob.command.ListDueInCommand;
+import bob.command.ListOnDateCommand;
+import bob.command.MarkCommand;
```

Comment:
should arrange your imports in alphabetical order if possible

Reply:
sflr yes, nice catch!